main.js

```diff
@@ -0,0 +1,140 @@
+let player = {
+  player1: {
+    name: "player1",
+    letter: "X",
+    winVal: 1 //when the abs(sum) of a row/col or diagonal is 3 the win condition is triggered
+  },
+  player2: {
+    name: "player2",
+    letter: "O",
+    winVal: -1
+  },
+  curentPlayer: "player1",
+  nextPlayer: "player2"
+};
+
+let start = document.getElementById("start1Player");
+let AIstart = document.getElementById("AIstart");
+let status = document.getElementById("turnIdentifier");
+let cells = document.getElementsByClassName("cell");
+let timer = document.getElementById("timer");
+let time = 0;
+
+AIstart.addEventListener("click", () => {
+  player.AI = true;
+  time = 0;
+  startGame();
+});
+
+start.addEventListener("click", () => {
+  time = 0;
+  startGame();
+});
+
+setInterval(gameTimer, 1000);
+function gameTimer() {
+  time += 1;
+  document.getElementById("timer").innerHTML = time;
+}
+
+function startGame() {
+  //resets the game
+  start.disabled = true;
+  start.hidden = true;
+  AIstart.hidden = true;
+  AIstart.disabled = true;
+  status.hidden = false;
+  timer.hidden = false;
+  player.curentPlayer = "player1";
+  player.nextPlayer = "player2";
+  player.player1.name = document.getElementById("player1Name").value;
+  player.player2.name = document.getElementById("player2Name").value;
+  status.innerHTML = player[player.curentPlayer].name + "'s turn";
+  for (let cell of cells) {
+    cell.onclick = () => turn(cell); //onClick better than EventListener in this case as i only want one event
+    cell.ticTacToe = 0;
+    cell.innerHTML = "";
+  }
+}
+
+function resetGame(){ //TODO is there a better way? looks fugly
```

Comment:
You could wrap it in different was, such as creating a `Game` class, and then creating a `new game()` on start, but somewhere in your code you're going to have to initialize these variables like this, and it'll look just as ugly.